packages/nameguard-sdk/src/index.ts

```diff
@@ -515,7 +544,7 @@ export class NameGuard {
 
   // TODO: Document how this API will attempt automated labelhash resolution through the ENS Subgraph.
   /**
-   * Inspects up to 250 names at a time with NameGuard. Provides a `ConsolidatedNameGuardReport` for each name provided in `names`, including:
+   * Inspects up to 250 names at a time with NameGuard. Provides `UninspectedNameGuardReport` if the `name` is uninspected; else returns a `ConsolidatedNameGuardReport` for each name provided in `names`, including:
```

Comment:
@djstrong I think it will be nice to update this to more formally refer to the new `inspected` field rather than the abstract idea of being inspected / uninspected.